gui/wxpython/main_window/frame.py

```diff
@@ -1948,8 +1962,6 @@ def _onMapDisplayFocus(self, notebookLayerPage):
         # moved from mapdisp/frame.py
         # TODO: why it is called 3 times when getting focus?
         # and one times when loosing focus?
-        if self.workspace_manager.loadingWorkspace:
-            return
```

Comment:
Just wondering, why do you remove this?

Reply:
I remove it because when loading the workspace file we need to focus on the map display page associated with the layer tree page. If these lines remained, we would focus on the last notebook page (so lmgr and notebook indexes would not be the same)